packages/sdds-icons/package.json

```
@@ -72,7 +72,9 @@
     "prepare": "npm run build",
     "build": "node ./scripts/build.mjs",
     "build:android": "node ./scripts/build-android.mjs",
-    "build:ios": "node ./scripts/build-ios.mjs"
+    "build:ios": "node ./scripts/build-ios.mjs",
+    "storybook": "storybook dev -p ${PORT:-7007} -c .storybook",
```

ESLint

[error] Unexpected template string expression.

(no-template-curly-in-string)